### PR TITLE
BUGFIX: Always bundle custom svgs instead of using `resource://` paths

### PR DESCRIPTION
**Pull request by mhsdesign**
Resolves: https://github.com/neos/neos-ui/issues/3616
Resolves: https://github.com/neos/neos-ui/issues/3712

See https://github.com/neos/neos-ui/issues/2092#issuecomment-2283775887 for an explanation

**What I did**

The custom icons are now included into the bundle instead of requested by path.

image

Also to unify things and simplify the build-stack, the Neos icon is now also bundled as plain svg instead of data-url:

image

Also the fallback image of the image editor will be fixed for cloud hosted resources now:

| before | after |
|--------|--------|
| image | image | 

Surprisingly i could not observer any pixel shifting. So this should be 100% the same visual look.

**How I did it**

**How to verify it**

### CODE REVIEW

File: packages/react-ui-components/src/Icon/resourceIcon.tsx [hunk 1]
```diff
@@ -13,6 +13,11 @@ export interface ResourceIconProps extends Omit<IconProps, 'theme'> {
     readonly theme?: ResourceIconTheme;
 }
 
+/**
+ * @deprecated please refrain from using resource paths
```
Comment (Sebobo) [inline packages/react-ui-components/src/Icon/resourceIcon.tsx:17]: The `Icon` component is still using the ResourceIcon when defining custom icons in yaml. What is the intended replacement?

**Review by Sebobo — CHANGES_REQUESTED** [review]
Works and looks great, but I don't see yet how we can deprecate the now old `ResourceIcon` if the new one does something else and from what I see doesn't support custom icons.

**Comment by mhsdesign** [comment]
As written in https://github.com/neos/neos-ui/issues/2092

> Case (3.) is more complicated, I'm afraid. The `<ResourceIcon/>` is a specialization of the (unfortunately public API) `<Icon/>`-component. We will need to deprecate its behavior, and (prior to that) find a way to allow for custom Icon URIs that can be resolved on the server side (we'll some sort of configurable mapping thing here).

My idea is to deprecate the behaviour but thats just for us and without real effect obviously.
To get rid of all usages we will - and i will plan do so - make the `NodeTypeSchemaBuilder` transform all `resource://` paths on the server. 

That will leave us with no more usages of client `resource://` paths. Wdyt?

**Review by Sebobo — APPROVED** [review]
Works on my machine <3

File: packages/neos-ui-editors/src/Editors/Image/resource/dummy-image.dataurl.svg
Comment (mhsdesign) [inline packages/neos-ui-editors/src/Editors/Image/resource/dummy-image.dataurl.svg:1]: This is now duplicated here and in `Neos.Neos` but yeah there is no way around this ... and it somehow belongs here imo

**Review by grebaldi — APPROVED** [review]
Looks & works very well :)

**Pull request merged**